source/aggregation-pipeline-builder.txt

```diff
@@ -73,11 +75,19 @@ Save a Pipeline
 Open a Saved Pipeline
 ---------------------
 
-1. Click the :guilabel:`Folder` icon at the top left of the
+1. Click :guilabel:`Pipeline` at the top left of the
```

Comment:
I think the original text `Click the :guilabel:`Folder` icon at the top left of the` was the correct one.
I just tested it and clicking on Pipeline does nothing.

Reply:
Done.